zeroex/ordervalidator/order_validator_js.go

```diff
@@ -43,3 +54,47 @@ func (s RejectedOrderStatus) JSValue() js.Value {
 		"message": s.Message,
 	})
 }
+
+const computeOptimalChunkBatchSize = 3
+
+// computeOptimalChunkSizes splits the signedOrders into chunks where the payload size of each chunk
+// is beneath the maxRequestContentLength. It does this by implementing a greedy algorithm which ABI
+// encodes signedOrders one at a time until the computed payload size is as close to the
+// maxRequestContentLength as possible.
+func (o *OrderValidator) computeOptimalChunkSizes(signedOrders []*zeroex.SignedOrder) []int {
+	chunkSizes := []int{}
+
+	payloadLength := jsonRPCPayloadByteLength
+	nextChunkSize := 0
+	wg := &sync.WaitGroup{}
+	wg.Add(1)
+	go func() {
+		batchIdx := 0
+		for _, signedOrder := range signedOrders {
+			if batchIdx%computeOptimalChunkBatchSize == 2 {
```

Comment:
This definitely deserves a comment. I'm assuming the intention is to only call `time.Sleep` some of the time instead of once for every signed order?